juicy-html.html

```diff
@@ -36,14 +38,26 @@
             }
         };
         /**
+         * Skips/disregards pending request if any
+         * @return {XMLHttpRequest} skipped XHR request, in fact just `this.pending`
+         */
+        JuicyHTMLPrototype.skip = function() {
```

Comment:
Since `_loadExternalFile` is "private", maybe this method should be called `_skip` (with an underscore)? Or maybe a fuller name `_abortLoadingExternalFile()`.